src/Repository/EventRegistrationRepository.php

```diff
@@ -41,6 +41,13 @@ public function findByUuidAndEvent(BaseEvent $event, array $uuids): array
         return $this->findBy(['event' => $event, 'uuid' => $uuids]);
     }
 
+    public function getByUuidAndEvent(BaseEvent $event, array $uuids): EventRegistrationCollection
```

Comment:
if the first argument is `Event` your method should be named as `getByEventAndUuids`